types/webvtt-parser/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/webvtt-parser",
+    "version": "2.2.9999",
+    "projects": [
+        "https://github.com/w3c/webvtt.js"
+    ],
+    "devDependencies": {
+        "@types/webvtt-parser": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Edoardo Morandi",
+            "githubUsername": "DefinitelyTyped"
```

Comment:
```suggestion
            "githubUsername": "dodomorandi"
```

Reply:
Wow, it definitely slipped through. Probably this happened because my `origin` is the original repo and the script took the wrong username. Thanks!